lib/internal/modules/esm/worker.js

```diff
@@ -18,30 +14,27 @@ parentPort.close();
 publicWorker.parentPort = null;
 publicWorker.workerData = null;
 const esmLoader = require('internal/process/esm_loader');
-const {defaultGetFormat} = require('internal/modules/esm/get_format');
+const { defaultResolve } = require('internal/modules/esm/resolve');
+const { defaultGetFormat } = require('internal/modules/esm/get_format');
+
+const defaultLoaderWorker = {
```

Comment:
this probably shouldn't be called a "worker"

Reply:
I have no strong preference apart from "anything but loader". `defaultLoaderHooks`? `defaultLoaderHandlers`? `defaultResourceSource`? `defaultResourceProviders`?

Reply:
I have a slight preference for "resource provider" after writing up that list.